### DISCUSSION TITLE
Form.Item use `value`/`onChange` with decendant instead of direct child?

### DISCUSSION THREAD

**Discussion by bombillazo** [ant-design/ant-design#42563] [Q&A] [answered]
Hello, is it possible to select what descendant component one wishes to use as the source input of a Form.Item?

Often an input component is wrapped by elements due to styling issues, but it seems that Form.Item searches for those props from the direct child. Is there a way to specify (perhaps via ID) which descendant element to get the values from?

**Comment by zombieJ** [comment, marked as answer]
For React it's hard to get by id since its real dom element which not align with VDM. You can create a HOC component wrap with the subcomponent binding that satisfy the requirement. But for the React way, the correct (maybe not the most convenient) way is customize component to pass props to target component.

**Reply by bombillazo** [reply]
Ok, just double-checking. We create a wrapper component for our inputs to bind/expose the required props. It's fairly easy, so it's not a big issue, but I wanted to know if we could save ourselves the trouble of doing that. Thanks!
